proxy/src/registry.rs

```diff
@@ -43,25 +44,25 @@ pub struct Metadata {
 pub enum Message {
     /// Issue a new org registration with a given id.
     #[allow(dead_code)]
-    OrgRegistration(OrgId),
+    OrgRegistration(Id),
 
     /// Issue an org unregistration with a given id.
-    OrgUnregistration(OrgId),
+    OrgUnregistration(Id),
 
     /// Issue a new project registration with a given name under a given org.
     ProjectRegistration {
         /// Actual project name, unique for org.
         project_name: ProjectName,
         /// The Org in which to register the project.
-        org_id: OrgId,
+        org_id: Id,
     },
 
     /// Issue a user registration for a given handle storing the corresponding identity id.
     UserRegistration {
         /// Globally unique user handle.
-        handle: UserId,
+        handle: Id,
         /// Identity id originated from librad.
-        id: registry::String32,
+        id: Option<String>,
```

Comment:
Is it allowed to register a user without this id?

Reply:
It's not required semantically by the spec